src/anm/resource_manager.js

```diff
@@ -168,13 +174,13 @@ ResourceManager.prototype.check = function() {
             if (errors[urls[u]]) error_count++;
```

Comment:
please move `u` definition from this loop in `var` block above it then, similarly to the cases before.